openmetadata-ui/src/main/resources/ui/src/components/Database/DatabaseSchema/DatabaseSchemaTable/DatabaseSchemaTable.tsx

```diff
@@ -192,7 +192,9 @@ export const DatabaseSchemaTable = ({
         if (!schemaDetails) {
           return;
         }
-        const updatedData = { ...schemaDetails, displayName: data.displayName };
+        const updatedData = data.displayName
+          ? { ...schemaDetails, displayName: data.displayName }
+          : { ...schemaDetails, displayName: undefined };
```

Comment:
We can improve here, Lets connect for that!

Reply:
Refactored the code.